### ISSUE TITLE
Fine-tuning details about TNT model

### ISSUE THREAD

**Issue by interstate50** [huawei-noah/CV-Backbones#76]
Hi, @iamhankai 

I am trying to replicate the results of the TNT paper that have been fine-tuned to datasets such as CIFAR-10 and CIFAR-100. Could you give details about hyper-parameters used (like batch size, learning rate etc.)

Thanks.

**Comment by Shaw95** [comment]
Hi, for CIFAR-10 and model TNT-B, you can fine-tune the model according to the following hyper-parameters. CIFAR-100 likewise.
--num-classes 10 --sched cosine  --opt adamw -j 8 --warmup-lr 1e-6 --mixup .8 --model-ema --model-ema-decay 0.99996 --aa rand-m9-mstd0.5-inc1 --color-jitter 0.4 --cutmix 1.0  --warmup-epochs 5 --opt-eps 1e-8 --repeated-aug --remode pixel --reprob 0.25 --amp --lr 5e-5 --weight-decay .0005 --drop 0 --drop-path .1 --b 16

**Comment by interstate50** [comment]
Hi, @Shaw95. Thank you for your reply. When the batch size is 16, what is the gpu num? 8 GPUs?

Otherwise, I have reproduced your TNT-Ti's result on ImageNet dataset (Top1-73.25). But when I transfer it to the cifar10 dataset, the results I got are terriable (w model-ema：**only 25**，w/o model-ema：**only 70**). I use the same training schedule to train the TNT-S model and can get the result 98.81 (w model-ema) on cifar10 dataset. Because your paper don't release the TNT-Ti model's performance on cifar10, could you give me your relevant results? Thank you very much.
